iphone/Classes/TiUIListView.m

```diff
@@ -578,10 +578,10 @@ - (void)updateSearchResults:(id)unused
 {
     if (searchActive) {
         [self buildResultsForSearchText];
-        [resultViewController.tableView reloadData];
     }
-    else {
-        [_tableView reloadData];
+    [_tableView reloadData];
+    if ([searchController isActive]) {
+        [resultViewController.tableView reloadData];
```

Comment:
Indentation. Also, when would `searchActive` be `NO` but `[searchController isActive]` be `YES`?